### PR TITLE
Update links to get Docker Desktop

### PR DESCRIPTION
**Pull request by StefanScherer**
### Proposed changes

This PR updates the links to get Docker Desktop for Mac and Windows.

### CODE REVIEW

File: docker-for-mac/faqs.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: Frequently asked questions (FAQ)
 
 ### How do I get the Stable or the Edge version of Docker Desktop?
 
-You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/?overlay=onboarding). To download the Edge version, see the [Edge release notes](/docker-for-mac/edge-release-notes/).
+You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-mac). To download the Edge version, see the [Edge release notes](/docker-for-mac/edge-release-notes/).
```
Comment (usha-mandya) [inline docker-for-mac/faqs.md]: Should we provide the stable URL here - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe?

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ For information about Edge releases, see the [Edge release notes](edge-release-n
 ## Docker Desktop Community 2.2.0.3
 2020-02-11
 
-> [Download](https://hub.docker.com/?overlay=onboarding)
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
```
Comment (usha-mandya) [inline docker-for-mac/release-notes.md]: This is the Stable release notes, so I guess we should point users to the Stable URL - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe? WDYT

File: docker-for-windows/faqs.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ title: Frequently asked questions (FAQ)
 
 ### How do I get the Stable or the Edge version of Docker Desktop?
 
-A. You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/?overlay=onboarding). To download the Edge version, see the [Edge release notes](/docker-for-windows/edge-release-notes/).
+A. You can download the Stable version of Docker Desktop from [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-windows). To download the Edge version, see the [Edge release notes](/docker-for-windows/edge-release-notes/).
```
Comment (usha-mandya) [inline docker-for-windows/faqs.md]: Should we point to the Stable version - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe?

File: docker-for-windows/release-notes.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ For information about Edge releases, see the [Edge release notes](edge-release-n
 ## Docker Desktop Community 2.2.0.3
 2020-02-11
 
-> [Download](https://hub.docker.com/?overlay=onboarding)
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
```
Comment (usha-mandya) [inline docker-for-windows/release-notes.md]: Same as above. This is the stable release notes, so guess we should point users to the stable download link - https://download.docker.com/win/stable/Docker%20Desktop%20Installer.exe?

**Comment by StefanScherer** [comment]
Updated the PR after a short discussion in standup.

**Pull request merged**